include/pybind11/cast.h

```diff
@@ -915,10 +937,15 @@ template <typename type> class type_caster_base : public type_caster_generic {
             nullptr, nullptr, holder);
     }
 
-    template <typename T> using cast_op_type = detail::cast_op_type<T>;
+    // allow moving of custom types
+    template <typename T> using cast_op_type = detail::movable_cast_op_type<T>;
 
     operator itype*() { return (type *) value; }
```

Comment:
Is this a typo? Shouldn't this be `return (itype*) value;`?

Reply:
Probably. I fixed the typo without getting any functional change.